### DISCUSSION TITLE
All connections disappear every 15 minutes

### DISCUSSION THREAD

**Discussion by mscreations** [netalertx/NetAlertX#499] [Q&A]
I'm having an issue where Pi-Alert is showing that all devices are disconnecting every 15 minutes. I can't seem to find anything in the logs to help troubleshoot what's going on. This is what the dashboard looks like when I login: 
![image](https://github.com/jokob-sk/Pi.Alert/assets/1254904/003cd79a-f68f-43d6-bf9f-efcba983863c)
I've had to disable all Down alerts as this causes the alerts to trigger every 15 minutes. I am scanning multiple VLANs as I described in #404. Any clue as to what I should try looking at to get this behavior to stop?

**Comment by mscreations** [comment]
Might be related to #463 as after implementing the workaround in that issue (changing arpscan schedule to */5 from */3) it has been steady. We shall see in the next couple days if this is corrected.

**Comment by jokob-sk** [comment]
The next version will display a warning if schedules are out-of-sync: 
![image](https://user-images.githubusercontent.com/96159884/277616866-e48d4e30-9e59-4e56-97c6-f268f99231ed.png)
